tests/cypress/e2e/pageBuilder/restrictions.cy.ts

```diff
@@ -90,5 +91,28 @@ describe('Page builder', () => {
             buttons.assertHasNoButtonForType('New Banner');
             buttons.assertHasNoButtonForType('New Event');
         });
+
+        it('should have nodetype and list limit restrictions on absolute areas', function () {
+            addNode({
+                name: 'absoluteArea',
+                parentPathOrId: '/sites/jcontentSite/home/area-main',
+                primaryNodeType: 'jnt:absoluteArea',
+                properties: [
+                    {name: 'j:allowedTypes', values: ['jnt:event']},
+                    {name: 'j:numberOfItems', value: 2},
+                    {name: 'j:level', value: 0}
+                ]
+            }).then(() => {
+                jcontent = JContent
+                    .visit('jcontentSite', 'en', 'pages/home')
+                    .switchToPageBuilder();
+                const absoluteArea = jcontent.getModule('/sites/jcontentSite/home/area-main/absoluteArea');
+                absoluteArea.get().find('[jahiatype="module"][type="absoluteArea"]').should('have.attr', 'listlimit', '2');
+                const buttons = absoluteArea.getCreateButtons();
+                buttons.assertHasNoButtonForType('New content');
+                buttons.getButton('New Event');
+                deleteNode('/sites/jcontentSite/home/area-main/absoluteArea');
```

Comment:
I think this should be done in an after clause. in case test fail for any reason stale data could remain during next tests execution.

you can wrap your test clause by a specific describe enclosing only your test, this way you can implement beforeEach, afterEach to create/clean the data.

Reply:
Thanks I did that.